index.html

```diff
@@ -44,6 +48,122 @@ <h3>I'm a software <br> developer </h3>
                 </a>
             </div>
         </section>
+
+        <section id="works">
+            <button class="arrowBtn " type="button"><i class="fa fa-angle-down"></i></button>
+            <h2 class="blueTitleText">My Recent <br> Works </h2>
+            <div class="divider"></div>
+
+            <ul class="grid-card-container">
+
+                <li class="project-card" >
+                    <div class="projectImage">
+                        
+                    </div>
+                    <article>
+                        <p class="cardTitle">Multi-Post Stories Gain+Glory</p>
```

Comment:
- [ ] Since this is a __title__, you're expected to use a `hx` tag, not a `p` tag